eng/Packages.props

```diff
@@ -24,6 +24,7 @@
     <PackageReference Update="System.CodeDom" Version="4.4.0" />
     <PackageReference Update="System.Collections.Concurrent" Version="4.3.0" />
     <PackageReference Update="System.Collections.Immutable" Version="1.5.0" />
+    <PackageReference Update="System.Configuration.ConfigurationManager" Version="4.5.0" />
```

Comment:
After recent experiences, these make me nervous. What are the chances we will go into another DLL mess in VS?

Reply:
Happily, this one is only referenced in .NET Core MSBuild: https://github.com/microsoft/msbuild/pull/4638/files#diff-4d3fc7780500131817fceb95ddcea3feR980

on Desktop we just use the Framework version.

Reply:
Fixed.